README.md

```diff
@@ -1,9 +1,16 @@
-# TO DO
+# tue_config
+
+## Special keys:
+- extend/extends: relative or absolute file path which is imported at that level
```

Comment:
Special 'tags' in yaml start with an exclamation mark. IMHO we should stick to this.

Reply:
Tags are different then keys. If I understand the document correct, a tag is more like type hinting. So the correct way would be `!include include: XX.yaml` However the yaml-cpp libary doesn't expose this information, just the `include: XX.yaml`. Using `!include: XX.yaml` is not accepted by the library as it will be parsed as: tag: !include, key: XX.yaml, value: "".